src/api/app/views/webui2/webui/package/_breadcrumb_items.html.haml

```diff
@@ -1,6 +1,6 @@
 = render partial: 'webui/project/breadcrumb_items'
 %li.breadcrumb-item.text-word-break-all
-  = link_to truncate(@package.name, length: 50), package_show_path(@project, @package)
+  = link_to truncate(@package.to_s, length: 50), package_show_path(@project, @package)
```

Comment:
I think we should use `@package_name` instead as we do in other controller actions. This is a general solution. I doubt that we will remember in some months why this `to_s` was needed.